ValidatorPanel.php

```diff
@@ -794,9 +794,15 @@ class ValidatorPanel extends Nette\Object implements Tracy\IBarPanel
 	 */
 	public function getTab()
 	{
-		return Html::el('span', ['title' => 'HTML Validator'])
-			->addHtml(Html::el()->setHtml(file_get_contents(__DIR__ . '/html_icon.svg')))
-			->addHtml(Html::el('span', ['class' => 'tracy-label'])->setText($this->errors ? (count($this->errors) . ' problems') : 'OK'));
+        $el = Html::el('span', ['title' => 'HTML Validator']);
+
+        // fix nette utils 2.3.10 - deprecated method Html::add()
+        $callMethod = method_exists($el, 'addHtml') ? 'addHtml' : 'add';
+
+        return $el
```

Comment:
move creating HTML elelement below `$callMethod`. Better to have it together

return Html::el('span', ['title' => 'HTML Validator'])